helm_deploy/court-case-service/templates/_envs.tpl

```diff
@@ -19,4 +19,28 @@ env:
         name: court-case-service-secrets
         key: APPINSIGHTS_INSTRUMENTATIONKEY
 
+  - name: DYNAMO_ACCESS_KEY
+    valueFrom:
+      secretKeyRef:
+        name: court-case-dynamodb-output
+        key: access_key_id
+
+  - name: DYNAMO_SECRET_KEY
+    valueFrom:
+      secretKeyRef:
+        name: court-case-dynamodb-output
+        key: secret_access_key
+
+  - name:
```

Comment:
you're missing the Env Var name here

Reply:
We don't need the ARN so I have removed it